test/whitelist.spec.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-unused-expressions */
+
+
+const faker = require('faker')
+const { getConfig } = require('../config')
+const XTokenTZ = require('./helpers/token')
+
+describe("adding accounts to WhiteListing category" ,()=>{
```

Comment:
Unusual capitalisation, please check `0.origination.spec.js`, `issuance.spec.js`, `transfer.spec.js` for naming tests